pkg/client/logs/logs.go

```diff
@@ -22,8 +22,10 @@ type (
 	}
 )
 
+// max number of items returned by Auth0 for each API call
+const ItemCountPerPage = 50
```

Comment:
Nitpick: I thought the max number here would be 100. I remember the go-auth0 client mentioning 100

Reply:
Unfortunately I didn't find anything about this, but for some reason I'm getting only `50` items. See https://github.com/tfadeyi/auth0-simple-exporter/pull/137#discussion_r1412346841

Reply:
The mystery is solved: https://github.com/tfadeyi/auth0-simple-exporter/pull/137#discussion_r1415373490